src/main/controllers/MediaAccountRejectionController.ts

```diff
@@ -0,0 +1,57 @@
+import {PipRequest} from '../models/request/PipRequest';
+import {Response} from 'express';
+import {MediaAccountApplicationService} from '../service/mediaAccountApplicationService';
+import {cloneDeep} from 'lodash';
+import moment from 'moment';
+
+const mediaAccountApplicationService = new MediaAccountApplicationService();
+
+export default class MediaAccountRejectionController {
+
+  public async get(req: PipRequest, res: Response): Promise<void> {
+    const applicantId = req.query['applicantId'];
+    const applicantData = await mediaAccountApplicationService.getApplicationByIdAndStatus(applicantId, 'PENDING');
+
+    if (applicantData) {
+      res.render('media-account-rejection', {
+        ...cloneDeep(req.i18n.getDataByLanguage(req.lng)['media-account-rejection']),
+        applicantData: applicantData,
+      });
+      return;
+    }
+
+    res.render('error', req.i18n.getDataByLanguage(req.lng).error);
```

Comment:
theres a lot of logic in all your controller methods, a good 70% of this can be in a service layer